.github/scripts/filter_test_configs.py

```diff
@@ -1,56 +1,89 @@
 #!/usr/bin/env python3
 
-import sys
-import re
 import json
 import os
+import re
+import sys
+import warnings
+from typing import Any, Dict, List, Set
+from urllib.request import urlopen
+
 import requests
-from typing import Any, Dict, Set, List
 import yaml
-import warnings
 
 PREFIX = "test-config/"
 
 # Same as shard names
-VALID_TEST_CONFIG_LABELS = {f"{PREFIX}{label}" for label in {
-    "backwards_compat",
-    "crossref",
-    "default",
-    "deploy",
-    "distributed",
-    "docs_tests",
-    "dynamo",
-    "force_on_cpu",
-    "functorch",
-    "inductor",
-    "inductor_distributed",
-    "inductor_huggingface",
-    "inductor_timm",
-    "inductor_torchbench",
-    "jit_legacy",
-    "multigpu",
-    "nogpu_AVX512",
-    "nogpu_NO_AVX2",
-    "slow",
-    "tsan",
-    "xla",
-}}
+VALID_TEST_CONFIG_LABELS = {
```

Comment:
Lintrunner ufmt change.  I have an action item to remove this hardcoded list later.